.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,67 @@
+name: Bug report
+description: Submit a bug report
+labels: ["type-bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+          # Bug report
+
+          ## New to Python?
+
+          The issue tracker isn't the right place to get help. Consider instead:
+
+            - reading the [Python tutorial](https://docs.python.org/3/tutorial/)
+            - posting on [Discourse](https://discuss.python.org/c/users/7)
+            - emailing [python-list](https://mail.python.org/mailman/listinfo/python-list)
+  - type: checkboxes
+    attributes:
+      label: Checklist
+      description: A bug in a third-party project (e.g. `pip` or `requests`) should be reported to that project's issue tracker, not CPython
+      options:
+        - label: I am confident this is a bug in CPython, not a bug in a third-party project
+          required: true
+        - label: |
+            I have searched the [CPython issue tracker](https://github.com/python/cpython/issues?q=is%3Aissue+sort%3Acreated-desc),
+            and am confident this bug has not been reported before
+          required: true
+  - type: textarea
+    attributes:
+      label: A clear and concise description of the bug
+      description: >
+        Tell us what happened.
+        Include a [minimal, reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) if possible.
+        Put any code blocks inside triple backticks.
+
+      value: |
+        ```python
+        # Add your code block here!
+        ```
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: CPython versions tested on
+      multiple: true
+      options:
+        - "3.8"
+        - "3.9"
+        - "3.10"
+        - "3.11"
+        - "3.12"
+        - "CPython main branch"
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Operating systems and architectures tested on
+      description: >
+        Feel free to leave this bit blank if the bug obviously isn't OS-specific.
+        Find out the architecture of your machine by running `python --version --version` on the command line.
```

``--version --version`` looks like a typo (I originally thought it was one!), even though it does produce the extended version information -- perhaps we could use the shorter version?:

```suggestion
        Find out the architecture of your machine by running `python -VV` on the command line.
```